src/EnergyPlus/PlantManager.cc

```diff
@@ -4531,6 +4541,20 @@ namespace PlantManager {
 		}
 
 	}
+	void
+	CheckOngoingPlantWarnings() 
+	{
+		int LoopNum;
+		for (LoopNum = 1; LoopNum <= TotNumLoops; ++LoopNum) {
+			//Warning if the excess storage time is more than half of the total time
+			if ( PlantLoop( LoopNum ).LoopSide( DemandSide ).LoopSideInlet_CapExcessStorageTime > PlantLoop( LoopNum ).LoopSide( DemandSide ).LoopSideInlet_TotalTime / 2) {
+				ShowWarningError("Plant Loop: " + PlantLoop( LoopNum ).Name + " Demand Side is storing excess heat the majority of the time." );
+			}
+			if ( PlantLoop( LoopNum ).LoopSide( DemandSide ).LoopSideInlet_CapExcessStorageTime > PlantLoop( LoopNum ).LoopSide( DemandSide ).LoopSideInlet_TotalTime / 2) {
```

Comment:
@rglazer2 This seems like a copy and paste error. From the warning message below I think it should be `SupplySide` otherwise this is the exact same if statement as above.